.github/workflows/run-danger.yml

```
@@ -7,7 +7,7 @@ on:
 jobs:
   dangermattic:
     if: ${{ (github.event.pull_request.draft == false) }}
-    uses: Automattic/dangermattic/.github/workflows/reusable-retry-buildkite-step-on-events.yml@v1.1.0
+    uses: Automattic/dangermattic/.github/workflows/reusable-retry-buildkite-step-on-events.yml@v1.1.1
```

@iangmaia WDYT about using v1 here?
```suggestion
    uses: Automattic/dangermattic/.github/workflows/reusable-retry-buildkite-step-on-events.yml@v1
```

This should always use the latest v1 version, and would save us from bumping the action with PRs like this. I guess it's safe as semantic versioning will prevent from bumping a breaking change version.

---

By the way, nice that Renovate catches even reusable workflows 👍

Hmm yeah if that works (indeed I think it should but I haven't tried it for reusable workflows) it should be safe by now (I wanted to be more strict in the beginning / developing).

---

@wzieba unfortunately, it failed: https://github.com/wordpress-mobile/WordPress-Android/actions/runs/9322366874

This would work only if we maintain a `v1` tag that we always keep updated by moving it to the latest `v1.x.x` 🤔

---

Right - I'll revert and merge this PR and we can discuss `v1` tag separately